### ISSUE TITLE
Staging Review finding: Heading at wrong level

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#101264]
![Image](https://github.com/user-attachments/assets/944d92c6-1203-4d2a-9780-0dab56027c80)

_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Veteran Facing Forms
**Product:** Direct Upload
**Feature:** Document Upload

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On each page of the form flow that has an info alert at the top of the page, the heading level for the alert's heading is H2. Since it falls beneath the step indicator's H2 and is a subsection of that part of the page, this doesn't seem like the right heading level.

Example: [/name-and-zip-code](https://staging.va.gov/form-upload/21-0779/name-and-zip-code), the "Name and zip code must match your form" heading in the alert is coded as an H2.

Screen reader users often navigate by heading, and heading levels provide information to them about the structure of the page. When headings aren't at the appropriate level or don't match the structure used elsewhere on the site, users may become confused or disoriented.

<details><summary>Link, screenshot or steps to recreate</summary>

![Personal information step with info alert.](https://github.com/user-attachments/assets/8bd5e56c-2305-438c-b128-9f93f9437fb0)

</details>

## Recommended action

Change the alert heading to H3.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jeana-adhoc** [comment]
[Issue being tracked here](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1991)

**Comment by jeana-adhoc** [comment]
This has been fixed!

<img width="778" alt="Image" src="https://github.com/user-attachments/assets/8161d09a-1145-4801-92ea-8b6b962b65b7" />
